docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,240 @@ created each run but it will be replaced on each subsequent run. If you would li
+to check whether the configuration they would provide for Snapshot process, are correct or not.
+
+Below is the code snippet for the validation function
+
+```scala
+import com.databricks.labs.overwatch.pipeline.Snapshot
+val sourceETLDB = "ow_bronze_snapshot_etl"
+val pipeline = "Bronze,Silver,Gold"
+val snapshotType = "Incremental"
+Snapshot.isValid(sourceETLDB,snapshotType,pipeline)
+```
+
+Upon running the above function you will get proper output whether the input parameters you would be using for Snapshot process,passed the validation check or not. One example of the same is as below:
+
+```
+ow_bronze_snapshot_etl is Overwatch Database and suitable for Snapshot
+Snapshot Type is Suitable for Snapshot Process. Provided SnapshotType value is Incremental
+Zone should be either Bronze,Silver or Gold. Provided Zone value is Bronze
+Zone should be either Bronze,Silver or Gold. Provided Zone value is Silver
+Zone should be either Bronze,Silver or Gold. Provided Zone value is Gold
+cloneLevel Type is Suitable for Snapshot Process. Provided cloneLevel value is DEEP
+Validation successful.You Can proceed with Snapshot process
+import com.databricks.labs.overwatch.pipeline.Snapshot
+res14: Boolean = true
+```
+
+### Restore Process
+Restore is the reverse process of Snapshot.In data world the term Restore is very common. It simply means if our current working space is corrupted then we can restore the current workspace from the snapshot location we have used earlier.
+
+Same like Snapshot Restore can be run by both Databricks Job and Databricks Notebook.
+
+##### Restore through Databricks Notebook
+
+The code snippet to run the Restore process is as below:
+```scala
+import com.databricks.labs.overwatch.pipeline.Restore
+val sourcePrefix = "/mnt/overwatch_playground/721-snap/full"
+val targetPrefix = "/mnt/overwatch_playground/721-snap/restore"
+Restore.process(sourcePrefix,targetPrefix)
+```
+We need below parameters to run the Restore process through Databricks Notebook:
+
+| Param          | Type   | Optional  | Description                                                                              |
+|----------------|--------|-----------|------------------------------------------------------------------------------------------|
+| sourcePrefix   | String | No        | Source ETL Path Prefix from where restore need to be performed.                          |
+| targetPrefix   | String | No        | Target ETL Path Prefix to where restore data would be loaded.                            |
+
+After Restore process is done the tables from sourcePrefix will be restored in the `target prefix/globalshare` location
+
+![Restore](/images/DataEngineer/restore_path.png)
+
+##### Restore through Databricks Job
+Restore process can also be configured using Databricks Job. For this we need to run RestoreRunner Class.
```

Comment:
Change to: "In cases where you need to periodically run the Restore process, you can configure a job using the RestoreRunner class."

Reply:
Done